config/tpcc_config_comdb2.xml

```diff
@@ -0,0 +1,43 @@
+<?xml version="1.0"?>
+<parameters>
+	
+    <!-- Connection details -->
+    <dbtype>comdb2</dbtype>
+    <driver>com.bloomberg.comdb2.jdbc.Driver</driver>
+    <DBUrl><![CDATA[jdbc:comdb2://localhost/tpcc?statement_query_effects=1&verify_retry=1]]></DBUrl>
```

Comment:
Do you need the `CDATA` part here?

Reply:
The '&' operator causes some XML parsing issue - The 'CDATA' makes XML escape the '&'